### DISCUSSION TITLE
absence of qSQI in zhao2018

### DISCUSSION THREAD

**Discussion by alexxony** [neuropsychology/NeuroKit#534] [Q&A] [answered]
**Question and context**
I checked the coding of zhao2018 and read the their paper. 
but I found qSQI is not applied. 
Can you explain why you did not apply ?
And How can i apply?

**Comment by zen-juen** [comment, marked as answer]
Hi @alexxony, we discussed it in our previous PR [here](https://github.com/neuropsychology/NeuroKit/issues/407). Briefly it seems like qSQI requires the use of 2 different peak detectors. 

The algorithm should be available here: [Zhao, Z., & Zhang, Y. (2018). "SQI quality evaluation mechanism of single-lead ECG signal based on simple heuristic fusion and fuzzy comprehensive evaluation". Frontiers in Physiology, 9, 727.](https://www.frontiersin.org/articles/10.3389/fphys.2018.00727/full)
